src/content/tutorials/fleet-management/cluster-management/cluster-autoscaler/index.md

```diff
@@ -0,0 +1,121 @@
+
+You can override these defaults in a `ConfigMap` named `cluster-autoscaler-user-values`.
+
+## Where is the user values ConfigMap
+
+The following examples assume the cluster you are trying to configure has an id of `123ab`.
+
+You will find the `cluster-autoscaler-user-values` `ConfigMap` on the organization namespace of your cluster:
+
+```text
+$ kubectl -n org-company get cm 123ab-cluster-autoscaler-user-values
+NAME                                       DATA      AGE
+123ab-cluster-autoscaler-user-values         0         11m
+```
+
+## How to set configuration options using the user values ConfigMap
+
+On the platform API, create or edit a `ConfigMap` named `<CLUSTER_ID>-cluster-autoscaler-user-values`
+in the workload cluster namespace:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: cluster-autoscaler
+  name: abc12-cluster-autoscaler-user-values
+  namespace: myorg
+data:
+  values: |
+    configmap:
+      scaleDownUtilizationThreshold: 0.30
+```
+
+## Configuration Reference
+
+The following sections explain some of the configuration options and what their defaults are. They show only the 'data' field of the ConfigMap for brevity.
+
+The most recent source of truth for these values can be found in the [values.yaml](https://github.com/giantswarm/cluster-autoscaler-app/blob/v1.30.3-gs1/helm/cluster-autoscaler-app/values.yaml) file of the `cluster-autoscaler-app`.
+
+### Scale down utilization threshold
+
+The `scaleDownUtilizationThreshold` defines the proportion between requested resources and capacity, which under the value `cluster-autoscaler` will trigger the scaling down action.
+
+Our default value is 65%, which means in order to scale down, one of the nodes has to have less utilization (CPU/memory) than this threshold.
+
+```yaml
+# 9.0.1 and greater
+data:
+  values: |
+    configmap:
+      scaleDownUtilizationThreshold: 0.65
+
+# 9.0.0 and below
+data:
+  scaleDownUtilizationThreshold: 0.65
+```
+
+### Scan Interval
+
+Define what interval is used to review the state for taking a decision to scale up/down. Our default value is 10 seconds.
+
+```yaml
+data:
+  values: |
+    configmap:
+      scanInterval: "100s"
+```
+
+### Skip system pods
+
+By default, the `cluster-autoscaler` will never delete nodes which run pods of the `kube-system` namespace (except `daemonset` pods). It can be modified by setting following property to `"false"`.
+
+```yaml
+data:
+  values: |
+    configmap:
+      skipNodesWithSystemPods: "false"
+```
+
+### Skip pods with local storage
+
+The `cluster-autoscaler` configuration by default deletes nodes with pods using local storage (`hostPath` or `emptyDir`). In case you want to disable this action, you need to set the following property to `"true"`.
```

Comment:
```suggestion
The cluster autoscaler by default deletes nodes with pods using local storage (`hostPath` or `emptyDir`). In case you want to protect these nodes from removal, you can to set the following property to true.
```